api/api-doc/storage_service.json

```diff
@@ -833,6 +833,43 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/repair_status/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Query the repair status and return when the repair is finished or timeout",
+               "type":"string",
+               "enum":[
+                  "RUNNING",
+                  "SUCCESSFUL",
+                  "FAILED"
+               ],
+               "nickname":"repair_async_status_until_finished",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"id",
+                     "description":"The repair ID to check for status",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type": "long",
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"timeout_ms",
```

Comment:
Setting ms timeouts on HTTP API is not very common, this could be easily seconds since if end state is reached it would return early anyway.

Reply:
Thanks for the feedback. I will make it seconds.